examples/example2.cpp

```diff
@@ -0,0 +1,445 @@
+/*
+ *  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+ *  SPDX-License-Identifier: MIT
+ *
+ *  Copyright (c) 2020-2021 Bhumit Attarde
+ *
+ *  Permission is hereby  granted, free of charge, to any  person obtaining a
+ * copy of this software and associated  documentation files (the "Software"),
+ * to deal in the Software  without restriction, including without  limitation
+ * the rights to  use, copy,  modify, merge,  publish, distribute,  sublicense,
+ * and/or  sell copies  of  the Software,  and  to  permit persons  to  whom the
+ * Software  is furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE  IS PROVIDED "AS  IS", WITHOUT WARRANTY  OF ANY KIND,  EXPRESS
+ * OR IMPLIED,  INCLUDING BUT  NOT  LIMITED TO  THE  WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR  A PARTICULAR PURPOSE AND  NONINFRINGEMENT. IN
+ * NO EVENT  SHALL THE AUTHORS  OR COPYRIGHT  HOLDERS  BE  LIABLE FOR  ANY
+ * CLAIM,  DAMAGES OR  OTHER LIABILITY, WHETHER IN AN ACTION OF  CONTRACT, TORT
+ * OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  OR THE
+ * USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include <iostream>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+//! [adding kitepp header]
+#include "kitepp.hpp"
+//! [adding kitepp header]
+
+namespace kc = kiteconnect;
+
+int main() {
+    try {
+
+        //! [initializing kite]
+        // kc::kite Kite(std::getenv("KITE_API_KEY"));
+        //! [initializing kite]
+
+        //! [obtaining login url]
+        // std::cout << "login URL: " << Kite.loginURL() << '\n';
+        // std::cout << "login with this URL and obtain the request token\n";
+        //! [obtaining login url]
+
```

Comment:
Are all these meant to be commented out? Is this for the docs?

Reply:
Yes